spec/views/users/index.html.erb_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+describe 'Post index page', type: :feature do
+  before :each do
+    @user = User.create(name: 'Marc', photo: 'Marc.jpg', bio: 'The One')
+    @post = Post.create(author: @user, title: 'Awakening', text: 'I choose the red pill')
+    @post_two = Post.create(author: @user, title: 'Training', text: 'I know kung fu')
+    @post_three = Post.create(author: @user, title: 'The Oracle', text: 'There is no spoon')
+    @post_four = Post.create(author: @user, title: 'Choice', text: 'I believe I can save Morpheus')
+    @post_five = Post.create(author: @user, title: 'Agent encounter', text: 'still only human')
+    @comment = Comment.create(post_id: @post, author_id: @user, text: 'time to fight')
+    @comment_two = Comment.create(post_id: @post_two, author_id: @user, text: 'time to fight')
+    visit user_posts_path(@user)
+  end
+
```

Comment:
Great job team 🤝
- As stated in the project requirements, kindly include tests that redirect to the user's show page when you click on a user.